src/x86.rs

```diff
@@ -266,6 +274,35 @@ impl X86Instruction {
         }
     }
 
+    /// Move to / from / between MMX (float mantissa)
+    #[allow(dead_code)]
+    #[inline]
+    pub const fn mov_mmx(size: OperandSize, source: X86Register, destination: X86Register) -> Self {
+        exclude_operand_sizes!(
+            size,
+            OperandSize::S0 | OperandSize::S8 | OperandSize::S16 | OperandSize::S32
+        );
+        if (destination as u8) & 16 != 0 {
```

Comment:
Can we write here for clarity that the bit mask tells us we have an MMX register?